Gemfile

```
@@ -3,43 +3,20 @@ source 'https://rubygems.org'
 gemspec
 
 group :test do
-  gem 'rake', '~> 10.0'
+  gem 'rake', '>= 10.0'
```

I think I'd pin it to 13
```suggestion
  gem 'rake', '~> 13.0'
```